### DISCUSSION TITLE
Adopting gene score model to bulk ATAC-seq data

### DISCUSSION THREAD

**Discussion by ammaralsheik** [GreenleafLab/ArchR#1866] [Feature Requests]
The gene score model that was optimized in ArchR performs really well and, such an approach is lacking for bulk ATAC-seq analysis, how possible is it to adapt this model to be used with bulk datasets?

**Comment by rcorces** [comment]
We do not have code for this at the moment but it is something that I've been asked for multiple times recently. In theory, this should work very well - it just doesnt fit in the current ArchR ecosystem because we have no support for bulk data.

**Comment by ammaralsheik** [comment]
Thank you, Ryan, Yes i definitely understand that it's not a direct fit. Hopefully someone will be able to adopt it or if we do we can share the code in the future.
